coalib/core/Bear.py

```diff
@@ -179,25 +185,11 @@ def maintainers_emails(cls):
                 if cls.MAINTAINERS_EMAILS == set() else
                 cls.MAINTAINERS_EMAILS)
 
-    @enforce_signature
-    def __init__(self, section: Section, file_dict: dict):
+    def __attrs_post_init__(self):
```

Comment:
@adhikasp I can keep the Docstring If you wish . Any tip @Makman2 @Udayan12167 ?